calendar-bundle/src/Resources/contao/languages/en/default.xlf

```diff
@@ -65,6 +65,9 @@
       <trans-unit id="MSC.eventPicker">
         <source>Events</source>
       </trans-unit>
+      <trans-unit id="MSC.eventOverview">
+        <source>Back to the event overview</source>
```

Comment:
I don't think this label makes sense. Whenever I have received a design from an agency that does not design for Contao, they often have such links and all simply are "Back". Since we don't know if the back link points to "an overview", I think we should keep calling it "back" and anyone with more specific need can still change the template.

Reply:
I disagree. We know exactly that it is a "back to the overview" link, because that is the main point of the PR. 😉

Reply:
The target module is called "event list". How do you know this is an "overview"? It could be the archive as well? I also know a lot of users that "abuse" the news module to list anything (e.g. projects of an architecture company). I sure don't want that link to be "back to news overview" if they are not news. "Go back" simply works for any and all cases 🤷

Reply:
I agree. It might also be called a "blog" for example.

Reply:
I hope you do realize that `MSC.eventOverview` is not a general label? It is a very specific label that is only used in the event module where we exactly know that we are going back to the event overview.

Reply:
I know. Would you like me to write the same on the `newsOverview` label, just because our examples were about the news module? 😂

Reply:
Personally I always find “Back” links hard to use on many websites because they often don’t tell me “Back to where exactly?”. I often look at which URL it points to, or use links from a breadcrumb navigation instead.

For me, “Back to the event overview” is way better than “Back”. And in cases where it doesn’t match with the terminology of the project, it gets changed to something like “Back to our Blog” or so, which is also way more useful than just “Back”.

So for me, tha fact that it has to be changed in some projects is a benefit (for the end user).

Reply:
> I also know a lot of users that "abuse" the news module to list anything (e.g. projects of an architecture company). I sure don't want that link to be "back to news overview" if they are not news.

Well, in this case it is their responsibility to also adjust the label, isn't it?